go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/signalfx/com_signalfx_metrics_protobuf
+
+go 1.14
```

Comment:
signalfx internal ingest services will still not be able to use this, as currently we support until 1.13.x

Reply:
Done.